examples/miscellaneous/miscellaneous_ex15/miscellaneous_ex15.C

```diff
@@ -0,0 +1,301 @@
+// The libMesh Finite Element Library.
+// Copyright (C) 2002-2020 Benjamin S. Kirk, John W. Peterson, Roy H. Stogner
+
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// Lesser General Public License for more details.
+
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+
+
+// <h1>Miscellaneous Example 15 - Laplace equation on unbound domain: Gravitational field computation</h1>
+// \author Hubert Weissmann
+// \date 2021
+//
+// This example uses second order infinite elements to solve the Laplace equation to compute the gravitational
+// field of three prism-shaped objects with same density. Probably the most important question that you always wanted
+// to study!
+//
+// Here infinite elements are used with frequency set to 0 so they can be used to describe the asymptotic behaviour
+// of the solution accurately.
+//
+#include <iostream>
+#include <fstream>
+// libMesh include files.
+#include "libmesh/getpot.h" // for input-argument parsing
+#include "libmesh/mesh.h"
+#include "libmesh/mesh_generation.h"
+#include "libmesh/linear_implicit_system.h"
+#include "libmesh/equation_systems.h"
+#include "libmesh/fe.h"
+#include "libmesh/elem.h"
+#include "libmesh/quadrature_gauss.h"
+#include "libmesh/dense_matrix.h"
+#include "libmesh/sparse_matrix.h"
+#include "libmesh/numeric_vector.h"
+#include "libmesh/dof_map.h"
+#include "libmesh/point_locator_tree.h"
+// for infinite elements:
+#include "libmesh/inf_fe.h"
+#include "libmesh/inf_elem_builder.h"
+//#include "libmesh/fe_interface.h"
+// for refinement:
+#include "libmesh/error_vector.h"
+#include "libmesh/uniform_refinement_estimator.h"
```

Comment:
You never ended up actually using URefEE, did you?  You're just doing uniform refinement in the loop below?  So we're not actually getting coverage of hanging nodes on uniform elements?

Reply:
yes. Thank you for pointing that out.
I wanted to make it as simple as possible; but of course I shouldn't minimize code coverage.